frontend/README.md

```diff
@@ -0,0 +1,26 @@
+# Frontend
+
+## Development
+
+1. Refer to `.env.example` file to create your own `.env` file
```

Comment:
is there a .env.example in the frontend folder?

Reply:
Oh yea, there isn't I not sure what env is needed in the frontend can you help me add it?

Reply:
yep I can add this